### PR TITLE
Removing Aio wrapper function to support ADC on MinnowBoard Turbot

### PR DESCRIPTION
**Pull request by uhofemei**
This removes the UPM ADC wrapper code.
Signed-off-by: Ulf Hofemeier <ulf.hofemeier@intel.com>

### CODE REVIEW

**Comment by uhofemei** [comment]
I was contemplating keeping the code just in case one wants to add an ADC to the MinnowBoard. I'll fix the typo soon.

Sent from my iPhone

On Mar 15, 2017, at 2:08 PM, Geoffroy Van Cutsem <notifications@github.com<mailto:notifications@github.com>> wrote:


Are you also planning to remove the folder and code in the helper-function folder?

Please also amend the README.md file accordingly.

Can you also fix the typo in the commit message in the updated PR?

**Comment by uhofemei** [comment]
Actually the README file in the ocf-server/js-server directory already contains a sentence explaining what the code in this directory is for. Do you want me to remove the directory?

**Comment by gvancuts** [comment]
I don't see the point in keeping code that's not used in the repo. I'd prefer to remove it all (inc. the reference that's in the `README.md`). If someone is interested, we can dig it up from the git history.

**Comment by uhofemei** [comment]
Fixed and I resubmitted the PR.

File: ocf-servers/js-servers/ambient_light.js [hunk 1]
```diff
@@ -28,10 +28,7 @@ var device = require('iotivity-node'),
 var mraa = '';
 try {
     mraa = require('mraa');
-    if(mraa.adcSupportedBits() == 0 ){
-        mraa.Aio =  require('./helper-function/mraa_MinnowBoard_ADC.js').Aio;
-    }    
-}
+    }
```
Comment (gvancuts) [inline ocf-servers/js-servers/ambient_light.js]: Nitpick: can you fix the indentation to be consistent with the rest of the file (the closing curly bracket should be aligned with the 't' in `try`)

File: ocf-servers/js-servers/gas.js [hunk 2]
```diff
@@ -29,10 +29,7 @@ var device = require('iotivity-node'),
 var mraa = '';
 try {
     mraa = require('mraa');
-    if(mraa.adcSupportedBits() == 0 ){
-        mraa.Aio =  require('./helper-function/mraa_MinnowBoard_ADC.js').Aio;
     }
-}
```
Comment (gvancuts) [inline ocf-servers/js-servers/gas.js]: Same comment as above re: indentation

**Comment by uhofemei** [comment]
Fixed.

**Pull request merged**